static/src/javascripts/projects/common/modules/commercial/braze/brazeBanner.tsx

```diff
@@ -106,13 +125,16 @@ const SDK_OPTIONS = {
 	devicePropertyAllowlist: [],
 };
 
-const getMessageFromBraze = async (apiKey, brazeUuid) => {
+const getMessageFromBraze = async (
```

Comment:
So this function used to set the `message` variable, and return a `Promise<boolean>`.
Now it just returns a `Promise<BrazeMessage>`

Reply:
Yeah exactly. Because of the way the banner picking works on frontend, and in particular `canShow` and `show`, the actual message returned from Braze is stored globally in a `message` variable in this file. That's still the case, but I've tried to reduce the places which have to know about that global state.